code/modules/surgery/limb_augmentation.dm

```diff
@@ -15,19 +17,21 @@
 
 /datum/surgery_step/add_limb
 	name = "replace limb"
-	implements = list(/obj/item/bodypart = 100)
+	implements = list(
+	/obj/item/bodypart = 100
+	)
 	time = 32
-	var/obj/item/bodypart/L = null // L because "limb"
+	var/obj/item/bodypart/L // L because "limb"
 
 
 /datum/surgery_step/add_limb/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	var/obj/item/bodypart/aug = tool
 	if(aug.status != BODYPART_ROBOTIC)
 		to_chat(user, "<span class='warning'>that's not an augment silly!</span>")
-		return -1

```

Comment:
hope you know how surgery steps work like down to the finest detail, -1 is not 0.

Reply:
I was under the assumption that -1 was 0, what does it parse as if not?